Resources/Prototypes/Store/presets.yml

```
@@ -35,4 +36,4 @@
     - SpellbookEquipment    #Battlemage Robes, Staff of Locker
     - SpellbookEvents       #Summon Weapons, Summon Ghosts
     currencyWhitelist:
-    - WizCoin
\ No newline at end of file
+    - WizCoin
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)